CONTRIBUTING.md

```diff
@@ -101,6 +101,15 @@ In your Jira instance, in the `Manage Apps` section, click on your App's button,
 
 After this is done, you should see your repos starting to sync in the App's dashboard.
 
+### Accessing Admin Endpoints
+
+There are some admin endpoints (see [/src/api/index.ts](/src/api/index.ts)) that require special permissions.
+
+To call these endpoints, you must:
+
+* be an admin in a special GitHub org
```

Comment:
You can mention the org here since it's in the code.

Reply:
yeah, I know. It's just one more place to change if it ever changes. I'll just link to the code, I guess.